utils/envvars/parser.go

```diff
@@ -72,13 +72,19 @@ func ParseEnvVars(envs []string) (envSettings *models.ChangeSettingsParams, errs
 			continue
 		}
 
+		var err error
 		// skip test environment variables that are handled elsewere with a big warning
```

Comment:
That's not what "elsewere" means, please update the comment. Or better yet, move `PERCONA_TEST_` warnings to the `default` block below. That way `PERCONA_TEST_DBAAS`can be handled by the same `switch`

Reply:
Ok